src/envoy/http/path_matcher/filter_config.h

```diff
@@ -100,9 +97,9 @@ class FilterConfig : public Envoy::Logger::Loggable<Envoy::Logger::Id::filter> {
 
   // Map from operation id to a PathParameterExtractionRule.
   // Only stores the operations that need path param extraction.
-  absl::flat_hash_map<
-      std::string,
-      ::espv2::api::envoy::v6::http::path_matcher::PathParameterExtractionRule>
+  absl::flat_hash_map<std::string,
+                      const ::espv2::api::envoy::v6::http::path_matcher::
+                          PathParameterExtractionRule*>
```

Comment:
Please use &

Reply:
Can't use references in map, they are not default constructable
https://stackoverflow.com/a/1543281/4402434